### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/lens/commit/baa0ffe04382bf5c0f6ced5c12f194bd46bca81a#commitcomment-52634475

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/lens/commit/21b5562db87f7980ed569c8880af0d36fa59ffe8

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: src/common/__tests__/catalog-category-registry.test.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ class TestCatalogCategory2 extends CatalogCategory {
 }
 
 describe("CatalogCategoryRegistry", () => {
-  it("should remove only the category registered when running the disopser", () => {
+  it("should remove only the category registered when running the disposer", () => {
```
Comment (jsoref) [inline src/common/__tests__/catalog-category-registry.test.ts:59]: Not sure

File: src/main/helm/helm-repo-manager.ts [hunk 2]
```diff
@@ -146,7 +146,7 @@ export class HelmRepoManager extends Singleton {
     return stdout;
   }
 
-  public static async addСustomRepo(repoAttributes : HelmRepo) {
+  public static async addCustomRepo(repoAttributes : HelmRepo) {
```
Comment (jsoref) [inline src/main/helm/helm-repo-manager.ts:149]: I think this was Cyrillic although I didn't check. n.b. not all instances of `addCustomRepo` in the files touched by this commit used the non-ascii `C`.

**Review by jsoref — COMMENTED** [review]
Note: corrections were almost entirely suggested by Google Sheets. I applied them and have tried to ensure they are reasonable.

The exceptions are:
* `Custom` (while Sheets didn't like `ustom`, it took a while for me to remember what it was)
*  `GitHub` because my tool sees this as `Git` + `Hub` and finds both satisfactory

**Comment by aleksfront** [comment]
@jsoref Can you rebase with latest `master` to resolve conflicts?

File: src/main/index.ts [hunk 3]
```diff
@@ -137,7 +137,7 @@ app.on("ready", async () => {
   /**
    * The following sync MUST be done before HotbarStore creation, because that
    * store has migrations that will remove items that previous migrations add
-   * if this is not presant
```
Comment (jsoref) [inline src/main/index.ts:140]: This was added between when the PR was originally made and this rebase

File: src/common/user-store/preferences-helpers.ts [hunk 4]
```diff
@@ -193,9 +193,9 @@ const hiddenTableColumns: PreferenceDescription<[string, string[]][], Map<string
   toStore(val) {
     const res: [string, string[]][] = [];
 
-    for (const [table, columnes] of val) {
-      if (columnes.size) {
-        res.push([table, Array.from(columnes)]);
```
Comment (jsoref) [inline src/common/user-store/preferences-helpers.ts:198]: As was this

**Comment by Nokel81** [comment]
I think that this should be run on each PR.

File: src/common/__tests__/catalog-category-registry.test.ts (hunk 1 above)
Comment (Nokel81) [reply src/common/__tests__/catalog-category-registry.test.ts:59]: This is the correct transform.

**Comment by jsoref** [comment]
Quite happy to make a PR for that. Do you want it included in this PR, or as a follow-up?

**Comment by Nokel81** [comment]
I think it should be in this one so that it is all contained.

**Comment by aleksfront** [comment]
@Nokel81 Suggesting to merge it now and made CI/CD improvements in upcoming PRs to speed up the process and prevent stale data.

**Review by aleksfront — APPROVED** [review]

**Comment by jsoref** [comment]
Sorry, I'll see if I can make a PR for this in a week or two.

**Pull request merged**